web/public/templates/profiles/general-info.dust

```diff
@@ -1,3 +1,13 @@
+<div ng-if="!editMode && !publicChampion">
+  <div class="alert alert-info" ng-if="parentProfile && profile.ownProfileFlag">
+    {@i18n key="Only your username and profile picture are publicly viewable. Other fields are for administration purposes only. "/}
+    <a ng-click="publicProfile()">Click to view your public profile</a>
+  </div>
+  <div class="alert alert-info" ng-if="!parentProfile && (profile.ownProfileFlag  || profile.myChild)">
```

Comment:
This div is exactly the same as the one above - could the logic in both ng-ifs be combined so that the html is only written once?